salt/_states/metalk8s_kubernetes.py

```diff
@@ -1582,6 +1582,72 @@ def customresourcedefinition_present(
     return ret
 
 
+def customresourcedefinition_absent(
+        name,
+        namespace='default',
+        body=None,
+        source=None,
+        **kwargs):
+    '''
+    Ensures that the named custom resource definitino is absent from
+    the given namespace.
+
+    name
+        The name of the custom resource definition.
+
+    namespace
+        The namespace holding the custom resource definition.
+        The 'default' one is going to be used,
+        unless a different one is specified.
+
+    body
+        The definition of the custom resource definition as a python dict.
+
+    source
+        A file containing the definition of the custom resource definition
+        in the official kubernetes format.
+    '''
+
+    ret = {
+        'name': name,
+        'changes': {},
+        'result': False,
+        'comment': ''
+    }
+
+    customresourcedefinition = __salt__[
+        'metalk8s_kubernetes.show_customresourcedefinition'](name, **kwargs)
+
+    if customresourcedefinition is None:
+        ret['result'] = True if not __opts__['test'] else None
```

Comment:
No, not salt good practice 

In test mode:
- Error when checking current status: `result=False`
- Nothing to do: `result=True`
- Some change needed: `result=None`

So here we should have `True` even if test mode enable